.github/workflows/ci.yaml

```
@@ -15,19 +15,117 @@ concurrency:
   group: ${{ github.workflow }} @ ${{ github.event.pull_request.head.label || github.head_ref || github.ref }}
   cancel-in-progress: true
 
-env:
-  DOCKER_IMAGE: hirosystems/${{ github.event.repository.name }}
-
 jobs:
+  api-lint:
+    strategy:
+      fail-fast: false
+      matrix:
+        suite: [ordinals]
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./api/${{ matrix.suite }}
+    steps:
```

If another push is done on a PRed branch that is running the CI currently, the CI related to the old CI should stop.

```suggestion
    steps:
      concurrency:
        group: ${{ github.workflow }}-${{ github.ref }}
        cancel-in-progress: true
```

---

We already have a concurrency setup in line 14 of the file